app/Entirety/entities/templates/entities/update.html

```diff
@@ -1,15 +1,23 @@
 {% extends '_base.html' %}
+{% load crispy_forms_tags %}
 {% load compress %}
 {% load static %}
 
 {% block title %}{% if update_entity %} Update Entity {% else %} Create New Entity {% endif %}{% endblock %}
 
 {% block content %}
     <span class="h3">{% if update_entity %} Update Entity {{ update_entity }} {% else %} Create New Entity {% endif %}</span>
+    {% if update_entity %}
+    {% else %}
+        <form method="post" action="">
+            {% crispy smart_data_model_form %}
```

Comment:
Could you please shorten the width of this dropdown within a div, what do you think? And can also icons from [here](https://icons.getbootstrap.com/?q=load) for Load button.

Reply:
And the same for Devices app.